test/unit/core/action-creator.ts

```diff
@@ -390,9 +394,11 @@ suite('ActionCreator', ({ expect, spy, stub }) => {
           const id = '1923';
           const state = { a: 'b' };
           const allMeta = 'hey';
-          const response = { c: 'd', records: [{ allMeta }] };
+          const record = { allMeta };
+          const response = { c: 'd', records: [record] };
           const detailsProduct = '10293';
           const dispatch = spy();
+          const emit = flux.emit = spy();
           const getStore = spy(() => state);
           const search = stub().resolves(response);
```

Comment:
put `const search = stub().resolves({ c: 'd', records: [record] })` bc response isnt used anywhere else. but it doesn't really matter